Parse/src/main/java/com/parse/ParseOkHttpClient.java

```diff
@@ -146,6 +156,124 @@ public ParseOkHttpClient(int socketOperationTimeout, SSLSessionCache sslSessionC
     return okHttpRequestBuilder.build();
   }
 
+  private ParseHttpRequest getParseHttpRequest(Request okHttpRequest) {
```

Comment:
We also have `Request getRequest(ParseHttpRequest parseRequest)` which is not a static method, to make it consistent I think non-static is better?